packages/react-native-worklets/src/shareables.ts

```diff
@@ -139,6 +139,18 @@ function makeShareableCloneRecursiveNative<T>(
     return cloneRemoteFunction(value, shouldPersistRemote);
   }
   if (isHostObject(value)) {
+    // https://github.com/facebook/react-native/commit/20dba39dab4ef85eb28659a89b19750cec3193a4
+    if (isHostObject(Object.getPrototypeOf(value))) {
+      const plainJSObject = {
+        ...Object.getPrototypeOf(value),
+      };
```

Comment:
Let's remove the comma so it fits in one line.
```suggestion
      const plainJSObject = { ...Object.getPrototypeOf(value) };
```
Alternatively, we can use `Object.assign({}, value)` directly to emphasize the fact that we're making a plain copy.